### PR TITLE
scalable umbraco for solution template

### PR DESCRIPTION
**Pull request by SunBuild**
### Changelog
updated the template to create two web apps ( one backend and one front end) 

### Description of the change
This is to meet the needs for azure solutions with CMS based apps for customer to deploy their web app scenarios.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@SunBuild__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sumuth). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: umbraco-cms-webapp-redis-cache/azuredeploy.json [hunk 1]
```diff
@@ -1,544 +1,579 @@
...
+                "Basic",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "SQL DB appServiceTier : Basic,Standard,Premium"
+            }
+        },
+        "NonAdminDatabaseUsername": {
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/azuredeploy.json]: parameter names need to be camelCase

File: umbraco-cms-webapp-redis-cache/azuredeploy.json [hunk 2]
```diff
@@ -1,544 +1,579 @@
...
+            "type": "string",
+            "metadata": {
+                "description": "Non-admin Database User. Must be Unique"
+            }
+        },
+        "NonAdminDatabasePassword": {
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/azuredeploy.json]: parameter names need to be camelCase

File: umbraco-cms-webapp-redis-cache/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Scalable Umbraco CMS template on Azure App Service Web Apps
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fumbraco-cms-webapp-redis-cache%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fumbraco-cms-webapp-redis-cache%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/README.md]: capital Azure. URLs are case sensitive

File: umbraco-cms-webapp-redis-cache/azuredeploy.parameters.json [hunk 4]
```diff
@@ -50,11 +23,11 @@
     "dbAdministratorLoginPassword": {
       "value": "GEN-PASSWORD"
     },
-    "dbNonAdminDatabaseUsername": {
-      "value": "umbracouser"
+    "NonAdminDatabaseUsername": {
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/azuredeploy.parameters.json]: This will change as well

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**